account_invoice_force_removal/models/account_move.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def unlink(self):
+        """Allow to remove moves related to invoices."""
+        types_allow = ["out_invoice", "out_refund", "in_invoice", "in_refund"]
```

Comment:
In case that limit to move types you can define this out of method to be easily extended.
Or the type list could be defined as system parameter.
Those are suggestions only, It isn't a required changes.